src/NUnitEngine/nunit.engine/Internal/SettingsStore.cs

```diff
@@ -89,7 +89,8 @@ public void LoadSettings()
             }
             catch (Exception ex)
             {
-                throw new ApplicationException("Error loading settings file", ex);
+                string msg = string.Format("Error loading settings {0}.", _settingsFile);
+                throw new ApplicationException(msg, ex);
```

Comment:
I think this should be an NUnitEngineException, which will get displayed without the stack trace.